modules/seqexec/web/client/src/main/scala/seqexec/web/client/reusability/package.scala

```diff
@@ -39,10 +39,15 @@ import seqexec.web.client.model.TabSelected
 import seqexec.web.client.model.SoundSelection
 import seqexec.web.client.model.GlobalLog
 import seqexec.web.client.circuit._
+import squants.Time
+import shapeless.tag.@@
 
 package object reusability {
   implicit def enumeratedReuse[A <: AnyRef: Enumerated]: Reusability[A] =
     Reusability.byRef
+  implicit def taggedInt[A]: Reusability[Int @@ A] =
+    Reusability.by(x => x: Int)
```

Comment:
I have never used shapeless's tag feature, but I wonder if this can this be generalized (ie: `Reusability[B @@ A]`) ?

Reply:
It should be assuming B has `Reusability` let me try

Reply:
This is a good aim, but after several tries I'm going to give up. We'd have to like with less generic versions for `Reusability` and `Pickler`